src/runtime/composables/useDevice.ts

```diff
@@ -1,6 +1,6 @@
-import { useNuxtApp } from '#app'
+import { useNuxtApp } from '#app';
```

Comment:
```suggestion
import { useNuxtApp } from '#app'
```
💅 
Semicolons don't seem to be required on this project 💪